Gemfile

```diff
@@ -6,10 +6,16 @@ source "https://rubygems.org"
 gemspec
 
 gem "rake", "~> 13.0"
-gem "rubocop", "~> 1.21"
 
 gem "pry", "~> 0.14.2"
 
 group :test do
   gem "simplecov", "~> 0.22.0", require: false
 end
+
+group :development, :test do
+  gem "rubocop", require: false
+  gem "rubocop-performance", require: false
+  gem "rubocop-rails", require: false
+  gem "rubocop-rspec", require: false
+end
```

Comment:
Let's remove this for now, and only focus on rubocop gem with a minimal rubocop.yml config file.

Reply:
Sure @keshavbiswa. Updated!